src/Pim/Bundle/CatalogBundle/tests/integration/Elasticsearch/AbstractIndexConfigurationIntegration.php

```diff
@@ -424,9 +181,19 @@ private function getProductIndexConfiguration()
                 ],
                 'settings' => [
                     'analysis' => [
-                        'normalizer' => [
-                            'lowercase_normalizer' => [
-                                "filter" => ["lowercase"],
+                        'normalizer'  => [
+                            'text_normalizer'    => [
+                                'type'        => 'custom',
+                                'filter'      => ['lowercase'],
+                                // TODO: Should be ["html_strip", "newline_pattern"] ?
+                                // But is not supported for now in ES 5.2 even though it is documented.
+                                // Those string transformations can be made at PRE_INDEX time in PHP.
+                                // Normalizer is an experimental feature.
+                                // see https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis-normalizers.html
+                                'char_filter' => [],
```

Comment:
It really seems complex to do on ES side. On PHP side, it's pretty simple and straightforward. If possible, put the complexity where people are more capable of managing it (in our case, PHP instead of ES scripts for example).